src/main/java/commands/FindCommand.java

```diff
@@ -0,0 +1,27 @@
+package commands;
+
+import storage.DataFile;
+import tasks.TaskList;
+
+public class FindCommand extends Command{
+    private final String keyWord;
+
+    public FindCommand(String keyWord) {
```

Comment:
Would it help with clarity if perhaps you can include JavaDoc headers for this FindCommand file?